extensions/interactions/MathExpressionInput/directives/math_expression_input_interaction_directive.html

```diff
@@ -1,5 +1,5 @@
 <div class="guppy-div"
-     style="width: 100%; height: 100px; padding: 5px;">
+     style="width: 100%; height: 100px; padding: 5px; overflow:auto; word-break: normal; line-break; word-wrap: normal">
```

Comment:
Add space after `overflow:`. Also, value of `line-break` is missing?
Add semicolon after `word-wrap: normal`.

Reply:
done.